### PR TITLE
Add volume level

### PR DESCRIPTION
**Pull request by Liborsaf**

### CODE REVIEW

File: custom_components/sony/media_player.py
Comment (alexmohr) [inline custom_components/sony/media_player.py:309]: Is the sleep really necessary?

File: custom_components/sony/media_player.py
Comment (alexmohr) [inline custom_components/sony/media_player.py]: should be debug or info level

**Review by alexmohr — CHANGES_REQUESTED** [review]
Thank you for the contribution. 
travis-ci does not seem to work anymore. I will look into migrating this to github ci in the next couple of days, so the linters can run before merging this :)

File: custom_components/sony/media_player.py
Comment (Liborsaf) [reply custom_components/sony/media_player.py]: Yeah, forgot to comment it. will push commit within 5 mins

File: custom_components/sony/media_player.py
Comment (Liborsaf) [reply custom_components/sony/media_player.py:309]: I couldn't find another solution, because when called without sleep, it will report old value

**Comment by Liborsaf** [comment]
Shouldn't this be also debug/info?
https://github.com/alexmohr/media_player.sony/blob/c834574c6ee2ead77ecfd175d47fbbc68b756114/custom_components/sony/media_player.py#L191
https://github.com/alexmohr/media_player.sony/blob/c834574c6ee2ead77ecfd175d47fbbc68b756114/custom_components/sony/media_player.py#L192

**Comment by alexmohr** [comment]
> Shouldn't this be also debug/info?
> 
> https://github.com/alexmohr/media_player.sony/blob/c834574c6ee2ead77ecfd175d47fbbc68b756114/custom_components/sony/media_player.py#L191
> 
> https://github.com/alexmohr/media_player.sony/blob/c834574c6ee2ead77ecfd175d47fbbc68b756114/custom_components/sony/media_player.py#L192

Yes, would make sense to change this as well. Thanks

**Comment by Liborsaf** [comment]
Is this okay to merge or does it need some tweaks?

Edit: Sorry, missed those linting issues :D Will fix now

**Comment by Liborsaf** [comment]
Will implement set volume into this MR as well

**Comment by Liborsaf** [comment]
I will always be forgetting about those linting things :D

**Comment by Liborsaf** [comment]
@alexmohr Wouldn't be better if linting doesn't require approval to run?

**Comment by alexmohr** [comment]
> @alexmohr Wouldn't be better if linting doesn't require approval to run?

Changed repo settings to `Require approval for first-time contributors who are new to GitHub ` which is the most liberal setting GitHub allows :)

**Comment by Liborsaf** [comment]
> > @alexmohr Wouldn't be better if linting doesn't require approval to run?
> 
> Changed repo settings to `Require approval for first-time contributors who are new to GitHub ` which is the most liberal setting GitHub allows :)

I see :)
This MR should be ready for merge, could you please review it?

**Pull request merged**
